src/interfaces/IPRBProxyRegistry.sol

```diff
@@ -224,18 +224,19 @@ interface IPRBProxyRegistry {
     ///
     /// Notes:
     /// - Installing a plugin is a potentially dangerous operation, because anyone can run the plugin.
-    /// - Plugin methods that have the same selector as {PRBProxy.execute} can be installed, but they can never be run.
+    /// - Plugin methods that have the same selectors as {PRBProxy.execute}, {PRBProxy.owner} and {PRBProxy.registry}
+    /// can be installed, but they can never be run.
     ///
     /// Requirements:
     /// - The caller must have a proxy.
     /// - The plugin must have at least one implemented method.
-    /// - There must be no method collision with any other installed plugin.
+    /// - There must be no method collision with any other plugin installed by the caller.
     ///
     /// @param plugin The address of the plugin to install.
     function installPlugin(IPRBProxyPlugin plugin) external;
 
-    /// @notice Gives or takes a permission from an envoy to call the provided target and function selector
-    /// on behalf of the caller's proxy.
+    /// @notice Sets the status of the envoy's permission to call the provided target on behalf of the caller's
+    /// proxy.
```

Comment:
The original wording sounds more idiomatic

Reply:
Fine with me.

Reply:
There are 2 issues with the original wording:
1) "Gives or takes a permission from" would mean that "from is used for both "gives" and "takes", which is wrong (see the discussion in another thread)
2) Saying that a permission is, say, given to an envoy for a target "**and function selector**" is redundant, because it's not like a permission could be given for a target, but not all of its functions, is it?